modules/client/examples/01-client/01-create-dao.ts

```diff
@@ -13,8 +13,8 @@ import {
   CreateDaoParams,
   DaoCreationSteps,
   DaoMetadata,
-  TokenVotingPluginInstall,
   TokenVotingClient,
+  TokenVotingPluginInstall,
```

Comment:
Since a lot of imports sorting is happening:

How about using `"@trivago/prettier-plugin-sort-imports": "^4.1.1",` (like we do here https://github.com/aragon/osx-plugin-template-hardhat/blob/bc914889cbd286b14d4c4a8a48f870a00ed8526e/package.json#L21) in another PR?

Reply:
We are using deno atm, and I ran a denofmt and made some changes, if we want to change to prettier or start using this for sorting I would say to leave it for another PR